### PR TITLE
Support swift package manager

### PR DESCRIPTION
**Pull request by akarsh**
Added package.swift file to make the library SwiftSpreadsheet supportable for SPM.

### CODE REVIEW

File: Package.swift [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+let package = Package(
+    name: "SwiftSpreadsheet",
+    products: [
+        .library(
+            name: "SwiftSpreadsheet",
+            targets: ["SwiftSpreadsheet"]),
+    ],
+    targets: [
+        // Targets are the basic building blocks of a package. A target can define a module or a test suite.
+        // Targets can depend on other targets in this package, and on products in packages which this package depends on.
```
Comment (stuffrabbit) [inline Package.swift]: Would be great if you could remove these comments.

File: Package.swift [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+        .target(
+            name: "SwiftSpreadsheet",
+            dependencies: [],
+            path: "SwiftSpreadsheet"
+        ),
```
Comment (stuffrabbit) [inline Package.swift]: Nit: Can you put the `),` on the line above, like in `.library`?

**Review by stuffrabbit — CHANGES_REQUESTED** [review]
Thanks a lot for contributing! Some minor comments below!

File: Package.swift (hunk 2 above)
Comment (akarsh) [reply Package.swift]: I agree I re-aligned the close bracket in this commit 
cc19641 .

File: Package.swift (hunk 1 above)
Comment (akarsh) [reply Package.swift]: Yeah, I forgot to remove auto-generated comments in this commit 
cc19641.

**Pull request merged**